dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -98,11 +100,38 @@ public static void main(String[] arguments)
       throws IOException, TemplateException, RepositoryException, URISyntaxException,
           ParseException, MavenRepositoryException {
     DashboardArguments dashboardArguments = DashboardArguments.readCommandLine(arguments);
-    Path output =
-        dashboardArguments.hasFile()
-            ? generate(dashboardArguments.getBomFile())
-            : generate(dashboardArguments.getBomCoordinates());
-    System.out.println("Wrote dashboard into " + output.toAbsolutePath());
+
+    if (dashboardArguments.hasVersionlessCoordinates()) {
```

Comment:
Should we report an error if someone passes more than one of (-c, -a, -f)?

Reply:
commons-cli throws `AlreadySelectedException` when more than one option group is set.